bootstrap/mod/src/main/java/org/geysermc/geyser/platform/mod/command/ModCommandSource.java

```diff
@@ -31,18 +31,19 @@
 import net.minecraft.network.chat.Component;
 import net.minecraft.server.level.ServerPlayer;
 import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
 import org.geysermc.geyser.GeyserImpl;
 import org.geysermc.geyser.command.GeyserCommandSource;
-import org.geysermc.geyser.platform.mod.GeyserModBootstrap;
 import org.geysermc.geyser.text.ChatColor;
 
 import java.util.Objects;
+import java.util.UUID;
 
-public class ModCommandSender implements GeyserCommandSource {
+public class ModCommandSource implements GeyserCommandSource {
 
     private final CommandSourceStack source;
 
-    public ModCommandSender(CommandSourceStack source) {
+    public ModCommandSource(CommandSourceStack source) {
         this.source = source;
     }
```

Comment:
load geyser locale like other CommandSource implementations?

Reply:
Unfortunately, the `CommandSourceStack` doesn't expose a locale - Floodgate-Fabric for example always fallbacks to en_US as of now: https://github.com/GeyserMC/Floodgate-Fabric/blob/master/src/main/java/org/geysermc/floodgate/util/FabricCommandUtil.java#L43